.github/workflows/cd-syft.yml

```diff
@@ -137,6 +137,36 @@ jobs:
             echo "github_release_version=$(python packages/grid/VERSION)" >> $GITHUB_OUTPUT
           fi
 
+      # Checkout to gh-pages and update helm repo
+      - name: Checkout to gh-pages
```

Comment:
MAYBE: We could move it last ?, as sometimes, before, we encountered an error, where we released helm charts, but there some errors in the build of containers.

Reply:
Another quick idea, we could leave it first initially, and merge the PR to check if it is working, because building containers, take a lot of time, we would have to wait a lot of time to check the same.

After it is successfully working we could move it to last

Reply:
Yeah sure we could move it to end. Wasn't sure if its used during container build step.

Reply:
Resolved by: 6b3c689baf8973f5021b1217f5557d58c362d02d